Sources/Automerge/actor_id.swift

```diff
@@ -1,5 +1,6 @@
 import AutomergeUniffi
 
+/// A type that represents a collaboration instance for managing replication within an Automerge document.
```

Comment:
This seems a little unclear to me, I'm not sure what "collaboration instance" refers to. Is this a known term in Swift? I've typically described the actor ID as representing a sequential thread of execution but il not sure that's a useful description either. The main thing is that no concurrent changes should be made by the same actor ID

Reply:
Fair enough - I don't know what to call the idea of different places that are syncing together, and have been leaning on that term to avoid confusing it was "a person", "a machine", or even "a tab in a browser". I'll go re-read the general [Automerge docs](https://automerge.org/docs/hello/) and see if they have a term of art already established there and try to replicate that.